src/components/Chat/index.tsx

```diff
@@ -1,43 +1,20 @@
 import React, { useEffect, useState } from 'react'
-import { AnimatePresence, motion } from 'framer-motion'
 import InkeepEmbeddedChat from './Inkeep'
 import { useChat } from 'hooks/useChat'
-import { IconDocument, IconRewind, IconX } from '@posthog/icons'
-import usePostHog from 'hooks/usePostHog'
-import { groupBy } from 'lodash'
-import dayjs from 'dayjs'
+import { IconDocument, IconX, IconCode } from '@posthog/icons'
 import { useApp } from '../../context/App'
 import { useWindow } from '../../context/Window'
-
-const ConversationHistoryButton = ({ onClick }: { onClick: () => void }) => {
```

Comment:
Why did we remove the conversation history? Not familiar with this bit of code, so curious!